tests/migrations/test_operations.py

```diff
@@ -1447,6 +1452,44 @@ def test_remove_index(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {'model_name': "Pony", 'name': "pony_test_idx"})
 
```

Comment:
It looks like most tests are using comments to separate sections rather than a blank line.